framework/util/show_coverage.pl

```diff
@@ -64,12 +64,14 @@ =head1 DESCRIPTION
     # Check for too many filenames
     pod2usage("$0: Too many files given.\n")  if (@ARGV > 1);
 
+my $expected_test_count = 30;
```

Comment:
Why is this number hard-coded?

Reply:
Also, why are lines 59-65 intented more than the variable declarations?  I'm not familiar with that style.